pro/src/new_components/VenueForm/VenueForm.tsx

```diff
@@ -10,6 +10,7 @@ import FormLayout from 'new_components/FormLayout'
 import { SubmitButton } from 'ui-kit'
 
 import { Accessibility } from './Accessibility'
+import Activity from './Activity/Activity'
```

Comment:
tout nos composant on leurs dossier, leurs index.ts et son importé comme suit: 
```suggestion
import { Activity } from './Activity'
```